src/main/java/seedu/weme/ui/ImportMemeCard.java

```diff
@@ -0,0 +1,70 @@
+package seedu.weme.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.weme.model.meme.Meme;
+
+/**
+ * An UI component that displays information of a {@code Meme}.
+ */
+public class ImportMemeCard extends UiPart<Region> {
+
+    private static final String FXML = "ImportMemeGridCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on MemeBook level 4</a>
+     */
+
+    public final Meme meme;
+
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private ImageView display;
+    @FXML
+    private Label id;
+    @FXML
+    private Label description;
+    @FXML
+    private FlowPane tags;
+
+    public ImportMemeCard(Meme meme, int displayedIndex) {
```

Comment:
@moziliar Should we have a BaseMemeCard and a MemeCardWithLikes? This is just memecard without likes